app/src/main/java/com/norvellium/tasky/core/validation/ValidatePassword.kt

```diff
@@ -0,0 +1,32 @@
+package com.norvellium.tasky.core.validation
+
+import android.content.Context
+import com.norvellium.tasky.R
+
+class ValidatePassword(context: Context) {
+
+    private val resources = context.resources
+
+    fun validate(password: String): ValidationResult {
+        if (password.isBlank()) {
+            return ValidationResult(false, resources?.getString(R.string.validation_error_password_empty))
+        }
+        // more than 8 characters
+        if (password.length < 8) {
+            return ValidationResult(false, resources?.getString(R.string.validation_error_password_length))
+        }
+        // at least one lowercase
+        if (!password.contains(Regex("[a-z]"))) {
+            return ValidationResult(false, resources?.getString(R.string.validation_error_password_no_lowercase))
+        }
+        // at least one number
+        if (!password.contains(Regex("[0-9]"))) {
+            return ValidationResult(false, resources?.getString(R.string.validation_error_password_no_number))
+        }
+        // at least one uppercase
+        if (!password.contains(Regex("[A-Z]"))) {
+            return ValidationResult(false, resources?.getString(R.string.validation_error_password_no_uppercase))
+        }
+        return ValidationResult(true)
+    }
```

Comment:
Same feedback here regarding context usage.

Also, IMO using list lambdas from the Kotlin standard lib make this a bit more readable, such as:
`val containsLowerLetter = password.any { it.isLowerCase() }`